### PR TITLE
#87 constants.js added.

### PR DESCRIPTION
**Pull request by VishalD51**
I have added constants.js and replace all hardcode value with constant defined.

---
Addresses: #87 

## Summary by CodeRabbit

- **Refactor**
  - Improved code readability and consistency in `BreadcrumbNavigation` and `HierarchySwitcher` components.
  - Enhanced error handling in breadcrumb navigation interactions.

- **New Features**
  - Integrated `defaultRegion` constant to standardize the initial state across navigation-related components.

- **Style**
  - Standardized code formatting and import statement styles in multiple components for better alignment with project code style guidelines.

- **Documentation**
  - No visible changes to end-user documentation.

### CODE REVIEW

File: frontend/src/components/BreadcrumbNavigation.js [hunk 1]
```diff
@@ -1,68 +1,79 @@
...
-        </Breadcrumbs>
-    );
+  return (
+    <Breadcrumbs aria-label="breadcrumb">
+      {breadcrumbItems.map((item, index) => (
+        <Typography
+          color="inherit"
+          key={index}
+          onClick={() =>
+            handleBreadcrumbClick(item.id, index, item.hasSubregions)
```
[the remark this answers is not in the document]
Comment (OhmSpectator) [reply frontend/src/components/BreadcrumbNavigation.js:68]: It's fixed in the latest main

File: frontend/src/components/HierarchySwitcher.js [hunk 2]
```diff
@@ -15,7 +17,7 @@ const HierarchySwitcher = () => {
         const hierarchies = await fetchHierarchies();
         setHierarchies(hierarchies);
       } catch (error) {
-        console.error("Error fetching hierarchies:", error);
+        console.error('Error fetching hierarchies:', error);
       }
```
[the remark this answers is not in the document]
Comment (OhmSpectator) [reply frontend/src/components/HierarchySwitcher.js:21]: It's out of the scope of this PR.

**Review by OhmSpectator — CHANGES_REQUESTED** [review]
The changes related to the introduction of the `defaultRegions` constant look fine. 

Nevertheless, avoid unnecessary code reformatting, please.

**Comment by OhmSpectator** [comment]
@coderabbitai review

**Comment by VishalD51** [comment]
> The changes related to the introduction of the `defaultRegions` constant look fine.
> 
> Nevertheless, avoid unnecessary code reformatting, please.

Thanks! @OhmSpectator , I will take care of it>

**Comment by VishalD51** [comment]
> > The changes related to the introduction of the `defaultRegions` constant look fine.
> > Nevertheless, avoid unnecessary code reformatting, please.
> 
> Thanks! @OhmSpectator , I will take care of it.

**Pull request closed without merging**